### ISSUE TITLE
Address issues discovered in February 2020 design QA

### ISSUE THREAD

**Issue by eddieantonio** [UAlbertaALTLab/cree-intelligent-dictionary#292]
We had a design QA session with Diane <time date="2020-02-19">a few days ago</time>. Here's a laundry list of issues to address:

 - [x] ~~"search for Cree or English" — placeholder text too big~~
 - [x] Search icon seems to be missing on production site #279 
 - [x] ~~Search icon on mobile should be left-aligned: offers affordance of "I can use this to search"~~
 - [x] ~~Search icon on tablet/desktop should be right-aligned: offers affordance of "I can click this to begin my search!"~~
 - [x] ~~prefix search should work~~ (future feature!)
 - [x] ℹ️ icon overflows its box
    - Use a `&nbsp;` between it and the (general) word class?
    -  needs margin
    - make (general) word class smaller?
 - [x] `<cite>` should be dark blue: `#182457`
 - [x] ~~Paradigm: fix issue with headings vs. column labels; headings should be left-aligned (blocked by #167) (see also: #193)~~
 - [x] ~~Paradigm table is misaligned in several places (blocked by #167)~~
 - [x] ~~there should be icons to explain the paradigm labels~~ (obsoleted by #363)
 - [x] row label & wordform should be `vertical-align: top`
 - [x] "show more" doesn't work — disable until we have the full paradigms
 - [x] `.paradigm-label--col { overflow-x: hidden }`
 - [x] More margin between head and its definitions.
 - [x] ~~There should be a circle around the ℹ️ icon when it's hovered/activated~~
 - [x] There should be a shadow under the box for the linguistic breakdown
 - [x] "Enable linguist mode" button should use `<button>` markup, but look like the "about" link
 - [x] update the copyright year — make it dynamic
 - [x] About page: margin between headings should be tighter to create a stronger visual hierarchy
 - [x] About page: links should have `font-weight: 700`
 - [x] logos should be laid out more carefully, using a 2x3 grid (#300)
 - [x] Search does not work on About page (crashing JavaScript) #280 
 - [x] ~~Search: preverbs should all display under a common heading~~
 - [x] decrease margin when there is no explicit definition
 - [x] change the colour of the 🔊 icon: `#286995` — shift in hue from the ℹ️ icon
 - [x] Tablet & desktop: the footer should use more horizontal space than vertical space
 - [x] There needs to be a clearer indication that a page is loading
 - [x] less margin between head and "form of {lemma}" text
 - [x] more margin between head and definition
 - [x] increase margin between `<hr>` and the content above
 - [x] Change from Zeplin mockup: recordings "bar" will be placed in same place as tablet/mobile, rather than appear on the same line as the head.
 - [x] Desktop: ℹ️ icon should be smaller
 - [x] vertically centre the 🔊icon

**Comment by eddieantonio** [comment]
I will do a handful of these and make a new PR, hopefully by this afternoon!

**Comment by eddieantonio** [comment]
Okay, I've picked all the low-hanging fruit in #293. What's left are things that we'll require more effort! I'm unassigning myself for now.

**Comment by aarppe** [comment]
@eddieantonio Is there still something crucial here that should be done? (We might separate that into an issue or few of their own.)

**Comment by eddieantonio** [comment]
@aarppe I've gone through the open tasks here and most of them are obsoleted by #862 and #363.

There is one bullet-point left, however:

>  - [ ] Search: preverbs should all display under a common heading

This is referring to the mockups in which preverbs and reduplication were all grouped together, rather than having one line per preverb as we currently have; see: 

![Mockup of a search page with preverbs and reduplication on one line](https://user-images.githubusercontent.com/2294397/127521725-096e376b-fee0-47e8-9877-834f53fc74c5.png)

 Should we group all prefixal elements on one line? Or is our current multi-line approach more favourable?

**Comment by aarppe** [comment]
@eddieantonio With the current approach, we want to keep the order of the prefixal elements, and reduplication and lexical preverbs can be interspersed (as reduplication will apply on the immediately following element but not generally over the whole wordform). So, the current solution is more faithful to the lexical meaning than the original UI proposal, and this one remaining subitem can be considered closed, and along with that, this entire issue.
